### PR TITLE
data: AdrianAcostaPR

### PR DESCRIPTION
**Pull request by AdrianAcostaPR**
Created AdrianAcostaPR.json file with information about my social media links and bio.

## Check List (Check all the applicable boxes) 

- [ ] My code follows the code style of this project.
- [ ] My change requires changes to the documentation.
- [ ] I have updated the documentation accordingly.
- [ ] All new and existing tests passed.
- [ ] This PR does not contain plagiarized content.
- [ ] The title of my pull request is a short description of the requested changes.

## Note to reviewers

<a href="https://gitpod.io/#https://github.com/EddieHubCommunity/LinkFree/pull/8356"></a>

### CODE REVIEW

File: data/AdrianAcostaPR.json [hunk 1]
```diff
@@ -0,0 +1,24 @@
+{
+  "name": "Adrian Acosta",
+  "displayStatsPublic": true,
+  "type": "personal",
```
Comment (Pradumnasaraf) [inline data/AdrianAcostaPR.json:4]:
```suggestion
```

**Review by Pradumnasaraf — CHANGES_REQUESTED** [review]
Hey,
Remove these two properties. By default, it's a `personal` type, and we no longer support public stats.

File: data/AdrianAcostaPR.json (hunk 1 above)
Comment (Pradumnasaraf) [inline data/AdrianAcostaPR.json:4]:
```suggestion
```

File: data/AdrianAcostaPR.json [hunk 2]
```diff
@@ -0,0 +1,24 @@
...
+  "bio": "Software Engineer (don't be fulled by the title, I'm still learning.). I work with Java on my 9-5, and I try to learn as much as I can from other technologies."
+  "avatar": "https://github.com/AdrianAcostaPR.png",
```
Comment (Pradumnasaraf) [inline data/AdrianAcostaPR.json:6]:
```suggestion
```

File: data/AdrianAcostaPR.json [hunk 3]
```diff
@@ -0,0 +1,24 @@
+{
+  "name": "Adrian Acosta",
+  "displayStatsPublic": true,
+  "type": "personal",
+  "bio": "Software Engineer (don't be fulled by the title, I'm still learning.). I work with Java on my 9-5, and I try to learn as much as I can from other technologies."
```
Comment (Pradumnasaraf) [inline data/AdrianAcostaPR.json:5]:
```suggestion
  "bio": "Software Engineer (don't be fulled by the title, I'm still learning.). I work with Java on my 9-5, and I try to learn as much as I can from other technologies.",
```

**Review by Pradumnasaraf — DISMISSED** [review]

**Comment by AdrianAcostaPR** [comment]
Thank you @kumarsonsoff3 and @Pradumnasaraf for the help! 
I apologize for the errors.  I kinda went on "Auto-pilot" with Eddie's video and didn't realize these errors.

**Comment by kumarsonsoff3** [comment]
It's okay, Adrian. It does happen! 😄

**Pull request merged**